.github/workflows/testing.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   drupal:
     name: Drupal ${{ matrix.drupal-core }} (PHP ${{ matrix.php-versions }})
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
@klausi For some reason it failing the pipeline for 11.0.x ~~ see: https://github.com/drupal-graphql/graphql/actions/runs/9996316696/job/27630424860?pr=1407#step:11:1257

Reply:
@klausi No, we can't use `ubuntu-latest` if we need to run PHPUnit tests on Drupal 11. Because as of today `ubuntu-latest` resolves to ubuntu 22.04. See [this](https://github.com/actions/runner-images?tab=readme-ov-file#available-images).
The Drupal 11 requires minimum SQLite 3.45 and ubuntu 24.04 (Ubuntu Noble Numbat) only supports SQLite 3.45 and above. See this: 
https://launchpad.net/ubuntu/+source/sqlite3
<img width="706" alt="Screenshot 2024-08-02 at 4 06 06 PM" src="https://github.com/user-attachments/assets/e13583a2-86b8-4410-b4d3-a66b40ac419d">

Reply:
See failing PHPUnit on Drupal 11 when ubuntu-latest is used: https://github.com/drupal-graphql/graphql/actions/runs/9995716069/job/27628492492#step:11:590

Reply:
I see, please add a code comment above like "# We cannot use ubuntu-latest right now as it still points to 22.04 and we need a newer database driver."